packages/analytics-browser/src/browser-client.ts

```diff
@@ -26,18 +27,32 @@ export class AmplitudeBrowser extends AmplitudeCore<BrowserConfig> {
       deviceId: oldCookies.deviceId ?? options?.deviceId,
       sessionId: oldCookies.sessionId ?? options?.sessionId,
       optOut: options?.optOut ?? oldCookies.optOut,
+      lastEventTime: oldCookies.lastEventTime,
     });
     await super.init(undefined, undefined, browserOptions);
 
-    // Step 3: Store user session in cookie storage
-    updateCookies(this.config, oldCookies.lastEventTime);
-
-    // Step 4: Install plugins
+    // Step 3: Install plugins
     await this.add(new Context());
     await this.add(new Destination());
 
     // Step 4: Track attributions
-    void this.trackAttributions();
+    if (!options?.attribution?.disabled) {
```

Comment:
Where is the old behavior preserved? Did we say if this is disabled, it keeps the old behavior?

Reply:
Old behavior does not exist in this package. Only new or nothing.